arch/arm64/boot/dts/qcom/sdm450-samsung-a6plte-r4.dts

```diff
@@ -0,0 +1,484 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
@vldly Are you sure we want to call this file `a6plte-r4.dts`? Not just `a6plte.dts`? Or do you know about different revisions of the same device that need different devicetree?